src/OrbitGl/App.cpp

```diff
@@ -2042,6 +2043,19 @@ void OrbitApp::LoadPreset(const PresetFile& preset_file) {
   });
 }
 
+void OrbitApp::ShowPresetInExplorer(const PresetFile& preset) {
+  const auto file_exists = orbit_base::FileExists(preset.file_path());
+  if (file_exists.has_error()) {
+    ERROR("Unable to find preset file: %s", file_exists.error().message());
+    return;
```

Comment:
This should probably be a user-facing error showing a text-box. I think you could use `SendErrorToUi` for that.

Reply:
Done! Thanks!